dev/app.js

```diff
@@ -338,5 +334,10 @@ search.addWidget(
   })
 );
 
+search.once('render', function() {
+  [...document.querySelectorAll('.smooth-search--hidden')]
```

Comment:
I modified the dev example to show the searchbox first and then render the whole interface when ready.